### PR TITLE
Update Contributing docs to include guidance around resolving comment…

### PR DESCRIPTION
**Pull request by radhikaj**
…s and using a browser extension for DCO sign off

Signed-off-by: Radhika Jandhyala <radhikaj@microsoft.com>

### CODE REVIEW

File: docs/Community/Contributing.md [hunk 1]
```diff
@@ -116,7 +116,9 @@ Please do:
 - **DO** ensure your correct name and email are on each commit.
 - **DO** address PR feedback in an additional commit(s) rather than amending the
   existing commits, and only rebase/squash them when necessary. This makes it
-  easier for reviewers to track changes.
+  easier for reviewers to track changes. After addressing comments, mark as resolved 
+  comments where you did exactly what the reviewer asked, and leave it up to the
+  reviewer to mark anything else as resolved.
```
Comment (CodeMonkeyLeet) [inline docs/Community/Contributing.md:121]:
```suggestion
  easier for reviewers to track changes.
- **DO** resolve only the comments that have been addressed exactly as requested by a
  reviewer. Leave it up to the reviewer to mark anything else as resolved.
```

**Comment by dthaler** [comment]
Looks like the build [failed](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/pipelines%2FAzure-Linux/detail/Azure-Linux/614/pipeline) due to issue #3099

**Comment by dthaler** [comment]
bors retry

**Pull request merged**